tools/travis-scripts/before-install.sh

```diff
@@ -41,22 +41,6 @@ function install_python_module_for_osx()
     sudo pip install Cheetah
 }
 
-function install_latest_python()
-{
-    python -V
-    eval "$(pyenv init -)"
-    pyenv install 2.7.14
-    pyenv global 2.7.14
-    python -V
-}
-
-function install_cmake_osx()
-{
-    cmake --version
-    brew upgrade cmake
-    cmake --version
-}
```

Comment:
Why don't have to update cmake now?

Reply:
Images xcode10.3, xcode11 have cmake 3.15, 3.15.1. Image Xcode9.3 has cmake 3.11

Reply:
Got it, so cmake is automatically updated when updating Xcode images? Right?

Reply:
Yes